screens/stores/StoreListScreen.js

```diff
@@ -18,12 +19,33 @@ class StoreListScreen extends React.Component {
     });
   };
 
+  updateSearch = search => {
+    this.setState({
+      search,
+      searchStores: this.state.stores.filter(this.filterStore(search))
+    });
+  };
+
+  filterStore(search) {
+    return store => {
+      return store.name.toLowerCase().includes(search.toLowerCase());
+    };
+  }
+
   render() {
+    const { search } = this.state;
+
     return (
       <StoreModal>
+        {/* Search bar */}
+        <SearchBar
```

Comment:
Nit: Should add a todo for styling, since SearchBar is from a library and not from `styled-components`

Reply:
Commented. Not sure how involved this ticket will be so am pushing it to the next sprint (also might be a good learning opp for new mem).